runtime/moonriver/src/governance/councils.rs

```diff
@@ -41,6 +41,9 @@ impl pallet_collective::Config<TreasuryCouncilInstance> for Runtime {
 	type WeightInfo = moonriver_weights::pallet_collective_treasury_council::WeightInfo<Runtime>;
 	type SetMembersOrigin = referenda::GeneralAdminOrRoot;
 	type MaxProposalWeight = MaxProposalWeight;
+	type DisapproveOrigin = EnsureRoot<AccountId>;
+	type KillOrigin = EnsureRoot<AccountId>;
+	type Consideration = ();
```

Comment:
Change to `FastGeneralAdminOrRoot`.

Reply:
Addressed in https://github.com/moonbeam-foundation/moonbeam/pull/3224/commits/83eb28e1e3a0c234c880d1d0977314ecd0a70d65